tests/x11/kontact.pm

```diff
@@ -52,7 +53,9 @@ sub run {
     # persist consistently as a process in the background causing kontact to
     # be "restored" after a re-login/reboot causing later tests to fail. To
     # prevent this we explicitly stop the kontact background process.
-    x11_start_program('killall kontact', valid => 0);
+    # matching 'genric-desktop' needs more time for LIVETEST
+    @tags = qw(desktop-runner-border desktop-runner-plasma-suggestions generic-desktop);
+    x11_start_program('killall kontact', target_match => \@tags, match_timeout => $match_timeout, valid => 0);
```

Comment:
why do you add the parameters for `target_match` together with `valid => 0`? That does not make sense to me as the parameters are only used if valid is a true value. I think something got confused during your testing